fdbclient/DatabaseConfiguration.h

```diff
@@ -32,16 +32,17 @@
 struct SatelliteInfo {
 	Key dcId;
 	int32_t priority;
+	int32_t satelliteDesiredTLogCount = -1;
 
-	SatelliteInfo() : priority(0) {}
+	SatelliteInfo() : priority(0), satelliteDesiredTLogCount(-1) {}
```

Comment:
```suggestion
	SatelliteInfo() : priority(0) {}
```
If you do the above this one is not needed ;)